projects/igniteui-angular/src/lib/query-builder/query-builder-functions.ts

```diff
@@ -0,0 +1,903 @@
+import { DebugElement } from '@angular/core';
```

Comment:
As this whole file is related to the tests, I suggest we add spec to its name. i.e. query-builder-functions.spec.ts.